### PR TITLE
Homework_2

### PR DESCRIPTION
**Pull request by shotvlad**

### CODE REVIEW

File: Task_1/Task_1.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿#include <iostream>
+
+using namespace std;
```
Comment (Stepka1709) [inline Task_1/Task_1.cpp:3]: test

**Review by stepka1709 — APPROVED** [review]

File: Task_4/Task_4.cpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            rangeFrom = number + 1;
+        }
+        else
+        {
+            cout << "The number of the computer is less." << endl;
+
+            rangeTo = number - 1;
+        }
+
+        cout << "Enter a number to guess the computer's number from " << rangeFrom << " to " << rangeTo << ": ";
```
Comment (Stepka1709) [inline Task_4/Task_4.cpp:55]: Test

File: Task_4/Task_4.cpp [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        cin >> number;
+    }
+    
+    return 0;
```
Comment (Stepka1709) [inline Task_4/Task_4.cpp:59]: test

**Review by stepka1709 — CHANGES_REQUESTED** [review]
rr

File: Task_2/Task_2.cpp [hunk 4]
```diff
@@ -0,0 +1,83 @@
+﻿#include <iostream>
```
Comment (VadzimAlkhimionak) [inline Task_2/Task_2.cpp:1]: There is so much repetitive code in this task

File: Task_2/Task_2.cpp [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    {
+        if (number1 == number2)
+        {
+            cout << "This triangle is isosceles and rectangular, where 3 side is the hypotenuse." << endl;
+        } 
+        else 
+        {
+            cout << "This triangle is rectangular, where 3 side is the hypotenuse." << endl;
+        }
+    }
```
Comment (VadzimAlkhimionak) [inline Task_2/Task_2.cpp:60]: repetitive code

**Review by VadzimAlkhimionak — COMMENTED** [review]
HI! Well done. Very good tasks.

**Pull request merged**